src/main/java/com/artipie/asto/memory/InMemoryStorage.java

```diff
@@ -47,16 +48,26 @@
     /**
      * Values stored by key strings.
      */
-    private final Map<String, byte[]> data = new HashMap<>();
+    private final NavigableMap<String, byte[]> data = new TreeMap<>();
```

Comment:
@olegmoz 
1. Why defining `NavigableMap` here. Just `Map` should be fine.
2. Please move the initializatiion into empty constructor.

Reply:
@fanifieiev good points, fixed it